src/pages/partners/affiliate-ib/_faq-data.tsx

```diff
@@ -43,25 +46,10 @@ const LocalizedLinkText = styled((props) => <LocalizedLink {...props} />)`
 
 const AffiliateGeneral = () => (
     <ItemContainer>
-        <Header
-            as="p"
-            type="paragraph-1"
-            mt="16px"
-            tabletL={{
-                type: 'paragraph-2',
-            }}
```

Comment:
We exatcly need to remove this behavior?
If isn't requirements for task, seems we should create new component with this kind of media response and use it instead of Header.
Like this:
const StyledHeader = styled(Header)`
      @media ${device.tabletL} {
        font-size: like paragraph-2;
        line-size: like paragraph-2;
        ...
    }
`